ports/mimxrt10xx/common-hal/microcontroller/Pin.c

```diff
@@ -224,3 +224,7 @@ bool common_hal_mcu_pin_is_free(const mcu_pin_obj_t* pin) {
 void common_hal_reset_pin(const mcu_pin_obj_t* pin) {
 //    reset_pin_number(pin->number);
 }
+
+void common_hal_never_reset_pin(const mcu_pin_obj_t* pin) {
+//    never_reset_pin_number(pin->port, pin->number);
```

Comment:
What happens to the display when the VM finishes? Does it stop working?

Reply:
I don't have the setup right now so can't check, but if we consider the port as WIP then I think it's fine for now, I want to spend more time with the SW soon so will implement this functionality :)

Let's merge as is?

Reply:
Works for me. We have https://github.com/adafruit/circuitpython/issues/2486 as a reminder to fix never reset.